pytorch_lightning/utilities/enums.py

```diff
@@ -256,3 +256,22 @@ def interactive_compatible_types() -> List["_StrategyType"]:
     def is_interactive_compatible(self) -> bool:
         """Returns whether self is interactive compatible."""
         return self in _StrategyType.interactive_compatible_types()
+
+
+class FaultTolerantTrainingMode(LightningEnum):
+
+    DISABLED = "disabled"
+    AUTOMATIC = "automatic"
+    MANUAL = "manual"
+
+    @property
+    def is_enabled(self) -> bool:
+        return self is not FaultTolerantTrainingMode.DISABLED
+
+    @property
+    def is_automatic(self) -> bool:
+        return self is not FaultTolerantTrainingMode.AUTOMATIC
+
+    @property
+    def is_manual(self) -> bool:
+        return self is not FaultTolerantTrainingMode.MANUAL
```

Comment:
should these be `self is <>` instead of `self is not <>` ?

Reply:
Done !